src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -818,3 +822,27 @@ func statefulSetKind() string {
 func (r *StatefulSetResource) fullConfiguratorImage() string {
 	return fmt.Sprintf("%s:%s", r.configuratorSettings.ConfiguratorBaseImage, r.configuratorSettings.ConfiguratorTag)
 }
+
+func (r *StatefulSetResource) getPreStopHook() *corev1.Handler {
+	var (
+		host = fmt.Sprintf("$(POD_NAME).%s", r.serviceFQDN)
+		port = r.pandaCluster.AdminAPIInternal().Port
+	)
+
+	cmd := strings.Join([]string{
+		"rpk debug info",
+		"grep -a 'redpanda.node_id'",
```

Comment:
I think this is not needed, we always set the node_id to index, you can see the code in configurator https://github.com/vectorizedio/redpanda/blob/bb522ea6e70f8243f6fccb735df5d975b3a626fb/src/go/k8s/cmd/configurator/main.go#L128

Reply:
oh, that's great!